mgr/src/main.rs

```diff
@@ -34,6 +35,11 @@ struct InstallArgs {
     /// Do not reboot after installation
     #[clap(long, action)]
     no_reboot: bool,
+
+    /// The mnemonic phrases and macaroons will automatically generate on remote server when kld first initialize.
+    /// This benefits when you own your remote server and can physically backup mnemonic phrases and macaroons without any copy through the internet
+    #[clap(long, default_value = "false")]
+    generate_secret_on_remote: bool,
```

Comment:
@paulmattei  The macaroons derive from mnemonic phrases, so I keep `generate_secret_on_remote` and not ` generate_mnemonic_on_remote`.